tests/utils.test.js

```diff
@@ -0,0 +1,45 @@
+const { convertToJson } = require("../scripts/utils");
+const {jsonString,yamlString,jsonObject,invalidString} = require("./fixtures/utilsData")
+
+describe('convertToJson', () => {
+  test('should return JSON object if input is valid JSON string', () => {
+    let result;
+    try {
+      result = convertToJson(jsonString);
+    } catch (error) {
+      result = error;
+    }
+    expect(result).toEqual(jsonObject);
+  });
+
+  test('should return JavaScript object if input is valid YAML string', () => {
+    let result;
+    try {
+      result = convertToJson(yamlString);
+    } catch (error) {
+      result = error;
+    }
+    expect(result).toEqual(jsonObject);
+  });
+
+  test('should return input if input is already an object', () => {
+    let result;
+    try {
+      result = convertToJson(jsonObject);
+    } catch (error) {
+      result = error;
+    }
+    expect(result).toBe(jsonObject);
+  });
+
+  test('should return null if input is invalid JSON and invalid YAML', () => {
+    let result;
+    try {
+      result = convertToJson(invalidString);
+    } catch (error) {
+      result = error;
+    }
+    expect(result).toBeNull();
+  });
```

Comment:
Here, the result is always null, no matter the `convertToJson` returns a null or not. The control flow is never getting into catch block.

The catch blocks in actual functions should throw an error like this

![image](https://github.com/asyncapi/website/assets/94157520/d22f367e-efeb-488f-be5e-1ab0d230403a)

After doing this, the catch block in your test would hit and the error would then be an actual error